scripts/flake_checker.py

```diff
@@ -214,7 +214,11 @@ def check_test_flakiness(output_lines: List[bytes], suite_name: str) -> bool:
         _print_color_message('Unable to convert json response: %s' % e)
         return False
 
-    if 'log' in report:
+    # If response doesn't have 'log' attribute, it will raise ValueError when
+    # changed to json format as FlakeReportDict. The error will be caught above
+    # and the case where the below if condition is false will never arise.
```

Comment:
It looks like we just use `FlakeReportDict` as the type annotation, which shouldn't have any effect at runtime (it just matters for MyPy). Therefore I don't see how we'd get a ValueError from calling `response.json()` when the response doesn't have the `log` attribute. Could you elaborate?

Reply:
Thanks @U8NWXD, I got that part wrong. Updated it now.